dist/api_data.py

```
@@ -1573,6 +1573,11 @@ def __ge__(self, other):
         Display page addresses, time windows, and page types as pages are verified, using the
         application's message handler, intended for debugging''',
         type='boolean'),
+    Config('dump_app_data', 'true', r'''
```

This is still on by default. This should default to hiding user's data.